ebuku.el

```diff
@@ -853,6 +853,49 @@ If an argument is excluded, get it from `ebuku-cache-default-args'."
       (setq tags (nconc (map-elt bookmark 'tags) tags)))
     (setq ebuku-tags (sort (seq-uniq tags) 'string-collate-lessp))))
 
+(defun ebuku-append-tag-to-bookmark ()
+  "Interactive function to append tag to bookmark."
```

Comment:
No need to use the phrase "Interactive function to ..." for interactive functions, so change to: "Append tag to bookmark at point."

Reply:
That makes it more clean, thanks!